shopfloor_mobile/static/wms/src/scenario/checkout.js

```diff
@@ -257,6 +271,14 @@ const Checkout = {
                 },
             };
         },
+        select_delivery_packaging_manual_select_options: function() {
+            return {
+                list_item_options: {
+                    loud_title: true,
+                    fields: [{path: "packaging_type"}, {path: "shipper_package_code"}],
```

Comment:
for `shipper_package_code` maybe better to show a label?

Reply:
done, same label than the backend